src/items/hit-location-item/hitLocationSheet.ts

```diff
@@ -49,6 +49,10 @@ export class HitLocationSheet extends RqgItemSheet<
     const itemData = this.document.data.toObject(false);
     assertItemType(itemData.type, ItemTypeEnum.HitLocation);
 
+    if (this.item.img === "icons/svg/item-bag.svg") {
```

Comment:
The foundry default icon can be found in this variable `foundry.data.ItemData.DEFAULT_ICON`